03-Raster-Calculations-In-R.Rmd

```diff
@@ -133,6 +149,25 @@ plot(DSM_HARV,
      main="Digital Surface Model (Elevation)\n Harvard Forest")
 ```
 
+Notice that the elevation range for these two maps (DSM and DTM) is different. 
+Why?  Is there any reason to question our data because of this elevational
```

Comment:
I don't understand this question. They are different products so the ranges should be different. What are you trying to get at here?